components/gitpod-cli/cmd/tasks-stop.go

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	supervisor_helper "github.com/gitpod-io/gitpod/gitpod-cli/pkg/supervisor-helper"
+	"github.com/gitpod-io/gitpod/supervisor/api"
+	supervisor "github.com/gitpod-io/gitpod/supervisor/api"
+	"github.com/manifoldco/promptui"
+	"github.com/spf13/cobra"
+)
+
+var stopTaskCmdOpts struct {
+	All bool
+}
+
+func stopTask(ctx context.Context, terminalClient supervisor.TerminalServiceClient, terminalAlias string) {
+	terminal, err := terminalClient.Get(context.Background(), &supervisor.GetTerminalRequest{Alias: terminalAlias})
+	if err != nil {
+		panic(err)
```

Comment:
I'm not sure about usage of panic, usually we are using logger with fatal exit. Although in cli it should not be so bad, could you check and align with how we do in other commands please

Reply:
Some commands use `panic` others use `fail`

```go
func fail(msg string) {
	fmt.Fprintln(os.Stderr, msg)
	os.Exit(-1)
}
```

I think they are similar and both OK for CLI, but we should consider using a single approach. I will create an issue for it.

Reply:
it seem even in this file we use `log.Fatalf` somewhere. Let's use `log` everywhere?